core/src/main/java/org/jclouds/Constants.java

```diff
@@ -325,7 +325,14 @@
     * </code>
     */
    public static final String PROPERTY_TIMEOUTS_PREFIX = "jclouds.timeouts.";
-   
+
+   /**
+    * Integer property. Default (32768).
+    * <p/>
+    * Buffer size for socket write.
```

Comment:
Explain here that this is only considered int he default java http driver

Reply:
closing this PR, reopened as https://github.com/apache/jclouds/pull/26 - thanks!